pkg/storkctl/clusterpair.go

```diff
@@ -321,13 +439,30 @@ func newCreateClusterPairCommand(cmdFactory Factory, ioStreams genericclioptions
 				return
 			}
 			storkops.Instance().SetConfig(conf)
-			_, err = storkops.Instance().CreateClusterPair(destClusterPair)
+			core.Instance().SetConfig(conf)
+
+			printMsg(fmt.Sprintf("Creating Secret and Backuplocation in destination cluster in namespace %v...", cmdFactory.GetNamespace()), ioStreams.Out)
+			_, err = core.Instance().CreateSecret(secret)
+			if err != nil {
+				util.CheckErr(err)
+				return
+			}
+			_, err = storkops.Instance().CreateBackupLocation(backupLocation)
 			if err != nil {
+				err := fmt.Errorf("unable to create backuplocation in destination. Err: %v", err)
 				util.CheckErr(err)
 				return
 			}
-			printMsg("ClusterPair "+clusterPairName+" created successfully on destination cluster", ioStreams.Out)
+			printMsg(fmt.Sprintf("Backuplocation %v added successfully on destination cluster in namespace %v\n", clusterPairName, cmdFactory.GetNamespace()), ioStreams.Out)
 
+			printMsg("Using PX-Service endpoints of source cluster to create clusterpair on destination cluster...", ioStreams.Out)
```

Comment:
in a similar fashion change the messages here

Reply:
Done